src/printer/CPPPrinter.cpp

```diff
@@ -242,6 +242,7 @@ void CPPPrinter::print(code::Code* prog) {
   fprintf(file, "#include \"random/Geometric.h\"\n");
   fprintf(file, "#include \"random/Poisson.h\"\n");
   fprintf(file, "#include \"random/InvGamma.h\"\n");
+  fprintf(file, "#include \"random/InvWishart.h\"\n");
```

Comment:
This line should be moved to line 270. These are all the distributions that do not need matrix.